mule-extensions-api-dsql/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>mule-extensions-api-parent</artifactId>
+        <groupId>org.mule.extensions</groupId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>mule-extensions-api-dsql</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+    <name>Mule Extensions API DSQL</name>
+    <description>API for Mule Extensions to add DateSense Query Language support.</description>
+    <url>https://github.com/mulesoft/mule-extensions-api</url>
+
+    <properties>
+        <antlrVersion>3.5</antlrVersion>
+        <formatterConfigPath>../formatter.xml</formatterConfigPath>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.mule.extensions</groupId>
+            <artifactId>mule-extensions-api</artifactId>
+            <version>${parent.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.antlr</groupId>
+            <artifactId>antlr-runtime</artifactId>
```

Comment:
this should either be provided or added to the whitelist. Regardless of what you do with antlr, this module itself needs to be added to the CE assembly and to the whitelist in both CE and EE